.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
 node_js:
     - "0.10"
-
+    - "0.12"
```

Comment:
These changes are unnecessary. Since T3 runs in the browser, there's no benefit to running the tests in multiple Node environments.